TFLite_detection_stream.py

```diff
@@ -75,37 +84,65 @@
 @app.route("/")
 def index():
 	# return the rendered template
-    return render_template("index.html")
+    # TODO: change password handling
+    if request.authorization and request.authorization.username == 'user1' and request.authorization.password == 'pass1':
+        return render_template("index.html")
+    return make_response('Could not verify!', 401, {'WWW-Authenticate' : 'Basic realm="Login Required"'})
```

Comment:
I agree with TODO item. We should authenticate user based on vault credentials in the near future.